layouts/partials/deprecation-warning.html

```diff
@@ -3,8 +3,9 @@
   <main>
     <div class="content deprecation-warning">
       <h3>
-        Documentation for Kubernetes {{ .Param "version" }} is no longer actively maintained. The version you are currently viewing is a static snapshot.
-        For up-to-date documentation, see the <a href="{{ .Site.Params.currentUrl }}">latest</a> version.
+	 Kubernetes {{ .Param "version" }}
+	 {{ T "deprecation_warning" }}
+	 <a href="{{ .Site.Params.currentUrl }}">{{ T "latest_version" }}</a>
```

Comment:
I'll let you decide, but you could leave the period outside of the string variable and put it after the </a> link tag.

Reply:
Nevermind. I see now that you need to use the Chinese period for the translation.